### PR TITLE
Add currentLocation and almaIdMMS

### PR DESCRIPTION
**Pull request by dr0i**
Results from hbz/lobid-resources#1257.

### CODE REVIEW

**Review by acka47 — COMMENTED** [review]
I adjusted the `rdfs:comment` for MMS ID in cb5b2c2 and added a comment wrt the ID of that property. Please decide by yourself whether an adjustment makes sense at this point.

File: lobid-vocab.ttl [hunk 1]
```diff
@@ -316,6 +328,20 @@ lv:hbzID
    rdfs:domain dct:BibliographicResource ;
    rdfs:range rdfs:Literal .
 
+# <http://purl.org/lobid/lv#almaIdMMS>
+
+lv:almaIdMMS
```
[the remark this answers is not in the document]
Comment (dr0i) [reply lobid-vocab.ttl:333]: As we have only one customer atm we will ask about that change and probably change it then. That should be decided tomorrow.
Posponed this issue.

**Comment by dr0i** [comment]
@hagbeck @UBMakla:
We would like to change the name of the `lv:almaIdMMS` to ` lv:almaMmsId` or just `lv:mmsID`.
1. Is it possible for you or makes this change to much trouble?
2. if 1) is ok - which one of the proposals do we take? I would pledge for `almaMmsId` because the source is more clear. What do you think?

**Comment by UBmakla** [comment]
That´s no problem. I would also prefer almaMmsId, but actually it doesn´t matter to us. Please just let us know which name you choose and from when the change is active.

**Comment by acka47** [comment]
I now adjusted the property URI in d53d280. @TobiasNx please review and merge if everythin is ok, Then you can go on and create an issue in lobid-resources to adjust the Alma data accordingly.

**Review by acka47 — APPROVED** [review]

**Review by TobiasNx — APPROVED** [review]
+1

**Pull request merged**